composer.json

```diff
@@ -25,7 +25,7 @@
   },
   "require-dev": {
     "phpunit/PHPUnit": "~4.6",
-    "squizlabs/php_codesniffer": "~2.0"
+    "squizlabs/php_codesniffer": "^2.3.1"
```

Comment:
Just curious, why this specific version and up? What changed specifically with 2.3.1 that is needed?